cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedElement.qll

```diff
@@ -752,7 +750,10 @@ newtype TTranslatedElement =
     not ignoreSideEffects(call) and
     // Don't bother with destructor calls for now, since we won't see very many of them in the IR
     // until we start injecting implicit destructor calls.
-    call instanceof ConstructorCall and
+    (
+    call instanceof ConstructorCall or
+    call instanceof DestructorCall
+    ) and 
```

Comment:
The comment above here is no longer correct, right? i.e., this part:
> // Don't bother with destructor calls for now, since we won't see very many of them in the IR
> // until we start injecting implicit destructor calls.